Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -1310,24 +1579,24 @@ def zeroAddRelabelling : ∀ x : PGame.{u}, 0 + x ≡r x
 theorem zero_add_equiv (x : PGame.{u}) : 0 + x ≈ x :=
   (zeroAddRelabelling x).equiv
 
-theorem leftMoves_add : ∀ x y : PGame.{u}, (x + y).LeftMoves = Sum x.LeftMoves y.LeftMoves
+theorem leftMoves_add : ∀ x y : PGame.{u}, (x + y).LeftMoves = (x.LeftMoves ⊕ y.LeftMoves)
```

Comment:
Again, I'd argue these stylistic changes would have been better off in another PR, but they're good changes so I once again approve of them.

Reply:
Already changed globally in #14934